core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -70,6 +70,32 @@ def validate_programme_risks(workbook: dict[str, pd.DataFrame]) -> list["TownsFu
         ]
 
 
+def validate_project_admin_gis_provided(
+    workbook: dict[str, pd.DataFrame]
+) -> list["TownsFundRoundFourValidationFailure"] | None:
+    """Validates that each project stating multiple locations contains a value for "GIS provided".
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the Round 4 submission.
+    :return: ValidationErrors
+    """
+    condition_broken = any(
+        workbook["Project Details"]["GIS Provided"][
+            workbook["Project Details"]["Single or Multiple Locations"] == "Multiple"
+        ].isna()
+    )
```

Comment:
nitpick: 
it seems standard in the other validations to pull the sheet out with:
Sheet = workbook[sheet_name]
Might it also make this section a tiny bit more readable?

Reply:
yeah good shout this is probably doing too much in one line